tests/test_timestamp_formats.py

```diff
@@ -0,0 +1,67 @@
+from pprint import pprint
+import pytest
+
+from datetime import datetime, timezone, timedelta
+from logmerger.timestamp_wrapper import TimestampedLineTransformer
+
+local_time = datetime.now().astimezone()
+local_tz = local_time.tzinfo
+
+def _test_timestamp_format_parsing(string_date, expected_datetime):
+    transformer = TimestampedLineTransformer.make_transformer_from_sample_line(
+        string_date
+    )
+
+    parsed_datetime, _ = transformer(string_date)
+    parsed_datetime = parsed_datetime.astimezone(timezone.utc)
+    pprint(parsed_datetime, width=200)
+    pprint(expected_datetime, width=200)
```

Comment:
pprint() and width=200 aren't necessary here, but I would like to add labels so we know which is which - also let's echo the given string_date:

```python
print(repr(string_date))
print("Parsed time  :", parsed_datetime)
print("Expected time:", expected_datetime)
```

Reply:
Sure thing. I updated with these.